ci-operator/step-registry/openshift/image-based/upgrade/seed/create/openshift-image-based-upgrade-seed-create-commands.sh

```diff
@@ -12,12 +12,10 @@ SSHOPTS=(-o 'ConnectTimeout=5'
   -o LogLevel=ERROR
   -i "${CLUSTER_PROFILE_DIR}/ssh-privatekey")
 
-PULL_SECRET_FILE="/var/run/pull-secret/.dockerconfigjson"
-PULL_SECRET=$(cat ${PULL_SECRET_FILE})
-BACKUP_SECRET_FILE="/var/run/ibu-backup-secret/.backup-secret"
-BACKUP_SECRET=$(cat ${BACKUP_SECRET_FILE})
-SEED_VM_NAME="seed"
 remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
+PULL_SECRET_FILE=$(cat ${SHARED_DIR}/pull_secret_file)
```

Comment:
Small nit, since we cat the file here, technically `PULL_SECRET_FILE` and `BACKUP_SECRET_FILE` is not referencing a file any longer, can we name it just `PULL_SECRET` or `BACKUP_SECRET` if we cat the file?

Reply:
The contents of `pull_secret_file` is actually the path to the file on the ec2 instance so it is in fact a filepath and not actually the pull secret itself. With that context, what do you think in terms of variable name? I was back and forth on what it should be called